syntax = "proto3";

package LiveProto;

// The greeting service definition.
service LiveMessager {
  // Sends a greeting
  rpc HandleJsonMsg (StringMsg) returns (StringMsg);
  rpc JsonMsgRouter(Empty) returns (stream StringMsg);
}

message Empty{}

 

// The request message containing the user's name.
message StringMsg {
	string type = 1;
	string jsonStr = 2;
  bytes pbdata = 3;
}


